ui/lib/core/addon/templates/components/linkable-item.hbs

```diff
@@ -0,0 +1,17 @@
+<div class="test" ...attributes>
+  {{#if @disabled }}
+    <div class="list-item-row linkable-item is-no-underline">
+      {{yield (hash content=(component 'linkable-item/content'))}}
+      {{yield (hash menu=(component 'linkable-item/menu'))}}
+    </div>
+  {{else}} 
+    <LinkedBlock 
+      @params={{array @link.route @link.model}} 
+      class="list-item-row linkable-item is-no-underline" 
+      data-test-auth-backend-link={{@link.model}}
+    >
+      {{yield (hash content=(component 'linkable-item/content'))}}
+      {{yield (hash menu=(component 'linkable-item/menu'))}}
+    </LinkedBlock>
+  {{/if}}
+</div>
```

Comment:
super small nit.  Add space here at the bottom of your hbs file, otherwise, you'll see the error icon.

Reply:
thank you! @Monkeychip - by space do you mean add a line so that the file is 18 lines long?